packages/altspacevr-extras/src/videoPlayerManager.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    Context,
+    User,
+} from '@microsoft/mixed-reality-extension-sdk';
+
+import {
+    ContextRPC,
+    UserRPC
+} from '@microsoft/mixed-reality-extension-sdk/built/rpc';
+
+type Video = { URL: string; basisTime: number };
+type VideoList = { [actorId: string]: Video };
+
+/**
+ * The main class of this app. All the logic goes here.
+ */
+
+export class VideoPlayerManager {
+    private _RPC: ContextRPC;
+    private userCount = 0;
```

Comment:
Unsure what `userCount` is for. When the user count drops to zero, the session is shut down. Are you protecting against something?